test/companies/companies_test.exs

```diff
@@ -84,9 +25,9 @@ defmodule Companies.CompaniesTest do
       assert %{name: "ZULU"} = Companies.get!("zulu")
     end
 
-    # test "does not retrieve deleted record" do
-    #  assert_raise Ecto.NoResultsError, fn -> Companies.get!(company.id) end
-    # end
+    test "returns nil if company is missing" do
+      refute Companies.get!("beta")
+    end
```

Comment:
Removed the commented out test and updated it with a test that tested the current functionality